.github/workflows/push_pr.yaml

```diff
@@ -135,6 +135,8 @@ jobs:
           go-version-file: 'go.mod'
       - name: Run unit tests
         run: make test
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
       - name: Setup Minikube
```

Comment:
Use the following notation instead:
```
uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
```
This is the secure way to pin a dependency. Once merged, check that Renovate updates the dependency to v3.1.5